olive/engine/engine.py

```diff
@@ -120,7 +128,13 @@ def __init__(
                     self.accelerator_specs.append(AcceleratorSpec(device, ep))
                     processed_ep.add(ep)
 
-        assert self.accelerator_specs, "No valid accelerator specified for target system."
+        assert self.accelerator_specs, (
+            "No valid accelerator specified for target system. "
+            "Please specify the accelerators in the target system or provide valid execution providers. "
+            f"Given execution providers: {self.execution_providers}. "
+            f"Current accelerators: {accelerators}."
+            f"Supported execution providers: {AcceleratorLookup.EXECUTION_PROVIDERS}."
```

Comment:
should here only show the providers mapped by the accelerators ?

Reply:
There is a case the accelerators is inferred from EPs. I think all mapping is good here.